### PR TITLE
files with recycler

### PR DESCRIPTION
**Pull request by alopesmendes**
Will display all files of a user. Need to click on the url to get a file.

Soon to be merge with main.

### CODE REVIEW

**Comment by amalekCoding** [comment]
Seems good to me, +1  👍 !  
Nice, but I prefer to wait for others reviews from the team before merging.

**Review by Jonakyll — APPROVED** [review]

**Pull request merged**
